cnf-tests/TESTLIST.md

```diff
@@ -90,26 +90,27 @@ The cnf tests instrument each different feature required by CNF. Following, a de
 
 | Test Name | Description |
```

Comment:
are these changes relevant to this patch?

Reply:
I needed the vendor update because RT kernel fix, but it brought some new tests, and CI requires that each test will have a description under the document, so I needed to add it

Reply:
Fair enough. Good enough for me. Let's hear from @fedepaol to make sure the network squad is OK with that.

Reply:
Cool, it means that it really works :+1: 
Yes, the idea is that whenever we add new tests, we provide a valid description and this is a way not to forget about it.